test/gurka/test_traffic.cc

```diff
@@ -50,9 +50,8 @@ TEST(Traffic, BasicUpdates) {
     printf("Make some updates to the traffic .tar file. "
            "Mostly just updates every edge in the file to 24km/h, except for one "
            "specific edge (B->D) where we simulate a closure (speed=0, congestion high)");
-    LiveTrafficCustomize cb_setter_24kmh = [&map](baldr::GraphReader& reader,
-                                                  baldr::TrafficTile& tile, int index,
-                                                  baldr::TrafficSpeed* current) -> void {
+    auto cb_setter_24kmh = [&map](baldr::GraphReader& reader, baldr::TrafficTile& tile, int index,
```

Comment:
these were conflicts. i saw someone made a typedef for the functor but i had already made it a const reference in my implementation and was able to just use `auto` for the lambda. i kept the change but i can revert it if someone has strong feelings